chat-widget/src/components/draggable/DraggableElementPosition.ts

```diff
@@ -0,0 +1,6 @@
+interface DraggableElementPosition {

```

Comment:
Every interface in this repo currently starts with an `I`. Might want to conform to the convention here